eng/Generation.psm1

```diff
@@ -25,6 +25,10 @@ function Invoke($command)
 function Invoke-AutoRest($baseOutput, $projectName, $autoRestArguments, $sharedSource, $fast)
 {
     $outputPath = Join-Path $baseOutput "Generated"
+    if ($projectName -eq "TypeSchemaMapping")
```

Comment:
Why do we need a special case for this project only?

Reply:
I am using this project to test when `outputPath` is not the default value. I tried to simply remove `outputPath`  in the command and use the values in readme.md instead, but I got quite a few errors for that and it would cost me more time to fix all the projects especially for the command when runnning with `-fast`, the changes needed seem tricky. So I chose the easy way to do it as I don't think we need more tests for `outputPath` not being the default value for now.